package.json

```diff
@@ -26,29 +26,27 @@
     "prepare": "husky || true"
   },
   "dependencies": {
-    "@headlessui/react": "^1.7.18",
+    "@headlessui/react": "^2.0.4",
     "@headlessui/tailwindcss": "^0.2.0",
-    "@heroicons/react": "^2.1.1",
-    "@hookform/resolvers": "^3.3.4",
-    "@react-oauth/google": "^0.12.1",
-    "@sentry/react": "^7.91.0",
-    "@sentry/vite-plugin": "^2.10.2",
-    "@tanstack/react-query": "^5.25.0",
-    "@tanstack/react-query-devtools": "^5.25.0",
-    "axios": "^1.6.7",
-    "date-fns": "^3.3.1",
+    "@heroicons/react": "^2.1.3",
+    "@hookform/resolvers": "^3.4.2",
+    "@sentry/react": "^8.4.0",
+    "@sentry/vite-plugin": "^2.17.0",
```

Comment:
Should we remove sentry dependencies since you commented the code to avoid having this installed if it is not needed?

Reply:
It could be, but not everyone knows which library needs installment.

Reply:
Perhaps removing and adding the libraries that are needed in the comment that says "Enable if needed" in `src/main.tsx`?